### ISSUE TITLE
Watson chatbot updates [priority]

### ISSUE THREAD

**Issue by vic-topcoder** [appirio-tech/connect-app#1294]
We need to add a disclaimer footer at the Chatbot powered by Watson project. Under the form body we need a line of text 
"IBM is receiving compensation from Topcoder for referring customers to IBM Watson Conversation Service."

This disclaimer should be only for the Chatbot. Please use a regular left-aligned label small (12px), $tc-gray-50.

![image](https://user-images.githubusercontent.com/8484061/32584292-64c504f4-c4ac-11e7-9525-7ab3353b41af.png)

**Comment by vic-topcoder** [comment]
@vikasrohit please add it and push to production ASAP - we're waiting on this to get the service going.

**Comment by vikasrohit** [comment]
Done. Added support for showing disclaimer at form bottom. @vic-topcoder I have shown the disclaimer at new project form and project edit form both.

**Comment by vic-topcoder** [comment]
The change is not in Dev, just production. Can we add the code to DEV as well please? Thanks for the quick job well done :)

**Comment by vic-topcoder** [comment]
@mtwomey @gondzo - can someone make sure we don't lose this from production? We need to have it in Dev as well.

**Comment by vikasrohit** [comment]
@vic-topcoder It was also in DEV. But we have another build, for next milestone, deployed on DEV which does not have these changes until we merge that branch into dev or vice versa. I have merged the dev into feature branch, so we should now have it dev as well.

However, I think due to some other css changes, it is coming right float while I cleared the float specifically. If you want it to be updated to production, let me know.
